### PR TITLE
Add SONiC VM support

### PR DESCRIPTION
**Pull request by adam-kulagowski**
This pull request will be followed up by PR in https://github.com/srl-labs/containerlab . The aim is to add SONiC VM support as a docker version of SONiC is severely limited/broken. The code itself is mostly a compilation of python/bash scripts from VSRX and FreeBSD directories. 

If anything is missing please let me know.

### CODE REVIEW

**Comment by adam-kulagowski** [comment]
Corresponding PR: https://github.com/srl-labs/containerlab/pull/2114

**Comment by hellt** [comment]
Thanks @adam-kulagowski !
I will try to get this reviewed beginning of next week

**Comment by hellt** [comment]
(you can close an eye on deepsource, I will remove it later on)

File: sonic/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This is the vrnetlab docker image for SONiC's VM.
+The scripts in this directory are based on FreeBSD and VSRX kinds.
+
+> Available with [containerlab](https://containerlab.dev) as `vr-sonic` kind.
+
+## Building the docker image
+
+Download the latest `sonic-vs.img.gz` image from https://sonic.software/
+unompress and place it in this directory. Rename the file to `sonic-vs-[version].qcow2` and reference that file in the `Makefile`.
```
Comment (hellt) [inline sonic/README.md:11]: since sonic.software is down, can you share with me the image you used for testing?

Comment (adam-kulagowski) [reply sonic/README.md:11]: It was working few days ago... anyway any generic qcow2 should work here as you already discovered :)

**Comment by hellt** [comment]
merged in #220 
thank you @adam-kulagowski

**Pull request closed without merging**